### PR TITLE
import the gpg keys from local files

### PR DESCRIPTION
**Pull request by jamesbeedy**
Add gpg keys from local files.

### CODE REVIEW

**Review by heitorPB — APPROVED** [review]
LGTM

File: slurm_ops_manager/slurm_deb_manager.py [hunk 1]
```diff
@@ -47,24 +47,22 @@ def _install_slurm_from_deb(self) -> bool:
...
-        subprocess.call(["apt-key", "add", key_two.as_posix()])
+        # Iterate over the two keys, adding them to the system.
+        for keyfile in ["04EE7237B7D453EC.key", "648ACFD622F3D138.key"]:
+            key = TEMPLATE_DIR / keyfile
+            subprocess.call(["apt-key", "add", key.as_posix()])
+
+        # Add the bullseye repo and run apt update.
+        Path("/etc/apt/sources.list.d/bullseye.list").write_text(
+            "deb http://deb.debian.org/debian bullseye main"
+        )
```
Comment (heitorPB) [inline slurm_ops_manager/slurm_deb_manager.py:65]: this should be in the previous place? Before installing the keyring?

File: slurm_ops_manager/slurm_ops_base.py [hunk 2]
```diff
@@ -18,14 +18,15 @@
 logger = logging.getLogger()
 
 
+TEMPLATE_DIR = Path(
+    os.path.dirname(os.path.abspath(__file__))) / 'templates'
```
Comment (heitorPB) [inline slurm_ops_manager/slurm_ops_base.py:22]: This fits in one line. Is there a reason to break in two?

**Review by heitorPB — CHANGES_REQUESTED** [review]

File: slurm_ops_manager/slurm_deb_manager.py (hunk 1 above)
Comment (jamesbeedy) [reply slurm_ops_manager/slurm_deb_manager.py:65]: https://wiki.debian.org/DebianKeyring - keyring is only used for storing gpg keys, we just need it installed before we add the keys.

**Comment by heitorPB** [comment]
On Thu, 29 Jul 2021 09:31:02 -0700
jamesbeedy ***@***.***> wrote:

> @jamesbeedy commented on this pull request.
> 
> 
> 
> > +        # Add the bullseye repo and run apt update.
> +        Path("/etc/apt/sources.list.d/bullseye.list").write_text(
> +            "deb http://deb.debian.org/debian bullseye main"
> +        )
> 
> https://wiki.debian.org/DebianKeyring - keyring is only used for
> storing gpg keys, we just need it installed before we add the keys. 
> 

But where does it come from? Ubuntu repos?

**Review by heitorPB — APPROVED** [review]
:+1:

**Pull request merged**